nemo_skills/pipeline/backends/slurm.py

```
@@ -0,0 +1,259 @@
+
+        For remote clusters, this checks SSH connectivity.
+        For local clusters, this checks if squeue is available.
+        """
+        import subprocess
+
+        ssh_tunnel = self.config.get("ssh_tunnel")
+
+        if ssh_tunnel:
+            # Check SSH connectivity
+            try:
+                result = subprocess.run(
```

Ruff (0.15.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)